action.yml

```
@@ -288,6 +300,11 @@ runs:
       gha_pull_request_change_detection = json.loads(
           os.environ['GHA_PULL_REQUEST_CHANGE_DETECTION']
       )
+      gha_sanity_tests = os.environ['GHA_SANITY_TESTS']
+      gha_sanity_skip_tests = os.environ['GHA_SANITY_SKIP_TESTS']
+      gha_sanity_allow_disabled = json.loads(
+          os.environ['GHA_SANITY_ALLOW_DISABLED']
+      )
```

Should this only allow `true` and `false`? Perhaps, this expectation could be made explicit here:
```suggestion
      ) is True
```